src/PushCommand.php

```diff
@@ -410,7 +410,7 @@ private function getGlobalVendorDir()
     {
         if (!$this->globalVendorDir) {
             $composer  = $this->getComposer(true);
-            $vendorDir = $composer->getConfig()->get('data-dir') . '/' . $composer->getConfig()->get('vendor-dir', Config::RELATIVE_PATHS);
+            $vendorDir = $composer->getConfig()->get('home') . '/' . $composer->getConfig()->get('vendor-dir', Config::RELATIVE_PATHS);
```

Comment:
Are you sure about the `home` config?
I've been able to find it by using `composer config --list --global`, but I did not find it in the documentation (https://getcomposer.org/doc/06-config.md).
Do you know if it's something official?

Reply:
I didn't find detailed description of the 'home' configuration in composer official document.
But I read the source code of composer (composer/SRC/composer/command/configcommand.PHP) and noticed that the following code exists from version 1.0.0 to the latest master version

$this - > config - > get ('home ').

So `home` configuration can be considered to be long-term supported.

I also did a test in the CentOS 7.6 environment. First, I deleted the local composer and `.config/composer/` directory under the user's root directory, and then reinstalled composer 1.1 and nexus push 0.1.7. I found that the execution of `composer nexus-push` failed. I need to change 'data dir' to 'home' to upload successfully. Here are the differences between the two configurations

[data-dir] /home/ym2048/.local/share/composer
[home] /home/ym2048/.config/composer

So I think the home configuration is reliable.

Reply:
good enough for me, thanks!
I'll merge and release a new version, thanks a lot!

Reply:
> @ym2048 I've released `0.4.2` with your fix in it

![image](https://user-images.githubusercontent.com/3700288/124053787-70877b80-da53-11eb-8632-9d9573688e40.png)